### ISSUE TITLE
Doesn't work with HDT 1.6.*

### ISSUE THREAD

**Issue by Vymrr** [CookieCompiler/Hearthstone-Collection-Tracker#2]
The witchwood numbers remain at 0 when I updated my collection.

**Comment by straussdd** [comment]
Same problem here. Looking into `%USERPROFILE%\AppData\Roaming\HearthstoneDeckTracker\CollectionTracker\Collection_Default.xml` shows this too:
```
<BasicSetCollectionInfo>
  <SetName>The Witchwood</SetName>
  <Cards />
</BasicSetCollectionInfo>
```

**Comment by CookieCompiler** [comment]
This bug is caused by HDT respectively HearthDB. At the moment they don't provide any data for the latest expansion. When calling `Hearthstone_Deck_Tracker.Hearthstone.Database.GetActualCards()` in [DataUpdaterV083](https://github.com/CookieCompiler/Hearthstone-Collection-Tracker/blob/master/Hearthstone%20Collection%20Tracker/Internal/DataUpdaters/DataUpdaterV083.cs#L49) there are only cards from Basic, Classic, Naxxramas, Goblins vs Gnomes, Blackrock Mountain, The Grand Tournament, League of Explorers, Whispers of the Old Gods, One Night in Karazhan, Mean Streets of Gadgetzan, Journey to Un'Goro, Knights of the Frozen Throne, Kobolds and Catacombs and Hall of Fame.

So the only thing we can do is waiting. Maybe one from @HearthSim can provide some information for this problem. 

/cc @azeier

**Comment by azeier** [comment]
Looks like the witchwood is missing from the `HearthDbConverter.SetDict` (fixed via https://github.com/HearthSim/Hearthstone-Deck-Tracker/commit/828d5e037f5b71181c58bb26b2f5ed5c95c362bc).

I'd recommend using `Card.CardSet instead of Card.Set`

**Comment by Vymrr** [comment]
awesome.. sounds like it can be fixed? Looking forward to the plugin being fixed..

**Comment by CookieCompiler** [comment]
I replaced Card.Set with Card.CardSet like @azeier recommended, but card count is still 0. Tested it together with HDT 1.6.4.

**Comment by azeier** [comment]
This succeeds for me ![gilneas](https://user-images.githubusercontent.com/7737910/38892220-f3533382-4286-11e8-966e-21405e509bd9.PNG)

**Comment by CookieCompiler** [comment]
This code works for me too. The problem is that the generated XML file has an empty card list for the expansion.
```
<BasicSetCollectionInfo>
  <SetName>The Witchwood</SetName>
  <Cards />
</BasicSetCollectionInfo>
```

**Comment by Vymrr** [comment]
Any updates?

**Comment by CookieCompiler** [comment]
I can't find any mistake in my code, so I wait for HDT 1.6.5 and will then check again.

**Comment by CookieCompiler** [comment]
Problem is fixed with HDT 1.6.5.
